data/de/vvo-trias.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "vvo",
+    "coverage": {
+        "realtimeCoverage": {
+            "area": {
+                "coordinates": [
+                    [ [ 11.8723081683, 51.6831408995 ], [ 15.0377433357, 51.6831408995 ], [ 15.0377433357, 50.1715419914 ], [ 11.8723081683, 50.1715419914 ], [ 11.8723081683, 51.6831408995 ] ]
+                ],
+                "type": "Polygon"
+            },
+            "region": [ "DE-SN" ]
+        },
+        "regularCoverage": {
+      		"region": ["DE"]
+    	}
+    },
+    "options": {
+        "endpoint": "http://efa.vvo-online.de:8080/std3/trias",
```

Comment:
Wow, it does not have working `https:`. 🤯

Reply:
I wrote myself a ticket to ask them, but didn't do it yet. For now let's document the reality.